src/amo/components/AddonReviewList.js

```diff
@@ -120,6 +120,7 @@ export class AddonReviewListBase extends React.Component {
           { authorName: review.userName, timestamp });
 
       const reviewBodySanitized = sanitizeHTML(nl2br(review.body), ['br']);
```

Comment:
It tidies up the next lines and makes it a bit more readable. I went back and forth on it and this was the nicest version.